.travis.yml

```diff
@@ -3,14 +3,14 @@ language: python
 matrix:
   include:
     # test with python 3.7 and latest flopy release available via conda
-    - env: ENV_FILE="ci/37-conda-flopy.yml"
+    - env: ENV_FILE="ci/38-conda-flopy.yml"
```

Comment:
`ci/37-conda-flopy.yml` should stay named as is, as it is supposed to test the package with python 3.7